DESCRIPTION

```diff
@@ -1,14 +1,15 @@
 Package: x13binary
 Type: Package
-Title: Provide the 'x13ashtml' Seasonal Adjustment Binary 
+Title: Provide the 'x13ashtml' Seasonal Adjustment Binary
 Version: 0.0.2
 Date: 2016-01-04
 Author: Dirk Eddelbuettel and Christoph Sax
 Maintainer: Dirk Eddelbuettel <edd@debian.org>
 Description: The US Census provides a seasonal adjustment program now
- called 'X-13ARIMA-SEATS' building on both earlier Census programs called
- X-11 and X-12 as well as the SEATS program by the Bank of Spain. Census
- offers both source and binary versions.
- .
- This package integrates these for use by other R packages. 
+    called 'X-13ARIMA-SEATS' building on both earlier Census programs called
+    X-11 and X-12 as well as the SEATS program by the Bank of Spain. Census
+    offers both source and binary versions.
+    .
+    This package integrates these for use by other R packages.
```

Comment:
I will re-adjust this to protest the roxygen2 obnoxiousness.

Hi @hadley, still a bug ...